resources/dicts/patrols/general/medcat.json

```diff
@@ -1057,7 +1680,7 @@
                 ]
             },
             {
-                "text": "p_l knows why this bothers s_c so much. A powerful connection to StarClan allows {PRONOUN/p_l/subject} to understand how their legacy would be told again and again to kits in the future. But p_l does {PRONOUN/p_l/poss} best to ease s_c's worries. To c_n, s_c would be a legend worth remembering one day. s_c brightens up at that remark, and the two go back to their herb hunt.",
+                "text": "p_l knows why this bothers s_c so much. A powerful connection to StarClan allows them to understand how their legacy would be told again and again to kits in the future. But p_l does {PRONOUN/p_l/poss} best to ease s_c's worries. To c_n, s_c would be a legend worth remembering one day. s_c brightens up at that remark, and the two go back to their herb hunt.",
```

Comment:
is the "them" and "their" referring to p_l or s_c?  is one of these cats meant to have the STAR skill?

Reply:
Hmm I don't remember writing this one. It might be an old patrol outcome that I moved.

p_l is the medcat, and medcats are generally seen as having a connection to starclan thus understanding their legacy? s_c is the warrior with them. so the them and their should be p_l's pronouns. (thinking through this outloud.)

Ill fix it up, I reworded it a bit so maybe it'll read cleaner?